src/geotableview.jl

```diff
@@ -34,9 +34,9 @@ function Base.values(v::GeoTableView, rank=nothing)
   r == R ? Tables.subset(𝒯, inds) : nothing
 end
 
-function constructor(::Type{GeoTableView{D,I}}) where {D<:AbstractGeoTable,I}
+function constructor(::Type{GeoTableView{G,I}}) where {G<:AbstractGeoTable,I}
   function ctor(domain, values)
-    geotable = constructor(D)(domain, values)
+    geotable = constructor(G)(domain, values)
```

Comment:
let's use D for the type, as if is referred to Data still. It is only a detail that data is represented with geotable